app/schedule/model.js

```diff
@@ -0,0 +1,20 @@
+import computed from 'ember-computed';
+import DS from 'ember-data';
+import moment from 'moment';
+
+export default DS.Model.extend({
+  start: DS.attr('date'),
+  end: DS.attr('date'),
+  objType: DS.attr(),
+  scheduleEventTitle: DS.attr(),
+  scheduleEventUrl: DS.attr(),
+  parentTitle: DS.attr(),
+  parentUrl: DS.attr(),
+
+  objIsSchedule: computed.equal('objType', 'ShowSchedule'),
```

Comment:
what's this? why is this part of the ember model? Is this to differentiate from an airing?